specification/azureactivedirectory/resource-manager/Microsoft.Aadiam/stable/2017-04-01/azureactivedirectory.json

```diff
@@ -37,6 +37,117 @@
     }
   },
   "paths": {
+    "/providers/microsoft.aadiam/privateLinkForAzureAd": {
+      "get": {
+        "tags": [
+          "privateLinkForAzureAd"
+        ],
+        "operationId": "privateLinkForAzureAd_List",
+        "x-ms-examples": {
+          "Gets the PrivateLinksForAzureAD": {
+            "$ref": "./examples/AzureADPrivateLinkPolicyList.json"
+          }
+        },
+        "description": "Operation to return the list of Private Links For AzureAD.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful request to get list of Private Links configuration objects For AzureAD.",
+            "schema": {
+              "$ref": "#/definitions/privateLinkForAzureAd"
+            }
+          },
+          "default": {
+            "description": "Error, failed to get list of Private Links For AzureAD.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/microsoft.aadiam/privateLinkForAzureAd/{policyName}/privateLinkResources": {
+      "get": {
+        "tags": [
+          "PrivateLinkResources"
+        ],
+        "description": "Gets the private link resources that need to be created for a policy of AzureAD.",
+        "operationId": "PrivateLinkResources_ListByPrivateLinkPolicy",
+        "x-ms-pageable": {
+          "nextLinkName": null
```

Comment:
>null [](start = 26, length = 4)

Can we add paging here too? (It's ok even if the service returns one page currently.)

Reply:
Added.